test/e2e/cleanup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+####################################################
+
+if [ -z "$SERVICE_PRINCIPAL_CLIENT_ID" ]; then
+    exit 1;
+fi
+
+if [ -z "$SERVICE_PRINCIPAL_CLIENT_SECRET" ]; then
+    exit 1;
+fi
+
+if [ -z "$TENANT_ID" ]; then
+    exit 1;
+fi
+
+if [ -z "$SUBSCRIPTION_ID_TO_CLEANUP" ]; then
+    exit 1;
+fi
+
+if [ -z "$EXPIRATION_IN_HOURS" ]; then
+    EXPIRATION_IN_HOURS=2
```

Comment:
good idea to echo this value so people looking at the console output know how far back we start